engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/TestEdition.scala

```diff
@@ -4,19 +4,36 @@ import nl.gn0s1s.bump.SemVer
 import org.enso.editions.{Editions, LibraryName}
 
 object TestEdition {
-  val librariesRequiredByTests: Seq[LibraryName] = Seq(
-    LibraryName("Standard", "Base")
+
+  val stdlibLibraries: Seq[LibraryName] = Seq(
+    LibraryName("Standard", "Base"),
+    LibraryName("Standard", "Test"),
+    LibraryName("Standard", "Table"),
+    LibraryName("Standard", "Database"),
+    LibraryName("Standard", "Image"),
+    LibraryName("Standard", "Geo"),
+    LibraryName("Standard", "Visualization"),
+    LibraryName("Standard", "Examples"),
+    LibraryName("Standard", "Searcher")
```

Comment:
This list is available in `project/Editions.scala`. I'm wondering if there would be a way to somehow get that information from there. Or maybe we can parse the edition?

Just feels like having this list in multiple places will end up with them getting out of sync sooner or later (probably someone will add a new stdlib package and it will not get appended here). If it's hard to extract this info from build metatada (probably would require something similar to the `buildinfo/Info.scala` generator), would be good to at least have a place in the docs or in a comment next to the list in `Editions.scala` that the lists should be kept up-to-date.